run.py

```diff
@@ -3,7 +3,9 @@
 
 
 protocol = pk.Protocol()
-model = protocol.generate_model()
+dictionary = protocol.read_config('pkmodel/config_file.txt')
```

Comment:
would be more elegant to generate protocol directly from read config, then not pass ie externalise the dictionary, see comment above